index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      rel="shortcut icon"
+      href="https://favicon-generator.org/favicon-generator/htdocs/favicons/2014-11-23/641ccf83998115288af7e77f866fb65c.ico"
+      type="image/x-icon"
+    />
+    <title>Apple</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      
+      <div class="image0">
```

Comment:
The CSS background should cover the whole page irrespective of the screen size. 
![image](https://user-images.githubusercontent.com/34653764/86066024-ed5ece80-ba68-11ea-9d17-035dfbe2dde2.png)